docroot/themes/humsci/humsci_basic/templates/components/pattern-alert.html.twig

```diff
@@ -0,0 +1,115 @@
+{# Override of Decanter's Core Template #}
```

Comment:
Is there a reason not to make this change in the [hs_layouts module template](https://github.com/SU-HSDO/suhumsci/blob/develop/docroot/modules/humsci/hs_layouts/patterns/alert/alert.html.twig)?

Reply:
I wasn't sure if you'd want us to alter it there, if it's not going to impact where this is used elsewhere I will make the change in that file and submit a change for it.